upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -335,13 +347,26 @@ func (e *InstanceTemplate) mapToGCE(project string, region string) (*compute.Ins
 		})
 	}
 
+	var accelerators []*compute.AcceleratorConfig
+	if e.GuestAccelerators != nil {
```

Comment:
I think this is a little better.
```suggestion
	if len(e.GuestAccelerators) > 0 {
```